test/ig/IGExporter.test.ts

```diff
@@ -0,0 +1,390 @@
+      expect(content.IG.excludexml).toEqual('Yes');
+      expect(content.IG.excludejson).toEqual('Yes');
+      expect(content.IG.excludettl).toEqual('Yes');
+      expect(content.IG.excludeMaps).toEqual('Yes');
+    });
+
+    it('should use the user-provided package-list.json when supplied', () => {
+      const pkgListPath = path.join(tempOut, 'package-list.json');
+      expect(fs.existsSync(pkgListPath)).toBeTruthy();
+      const content = fs.readJSONSync(pkgListPath);
+      expect(content).toEqual({
+        'package-id': 'sushi-test',
+        title: 'FSH Test IG',
+        canonical: 'http://hl7.org/fhir/sushi-test',
+        introduction: 'Custom intro',
+        list: [
+          {
+            version: 'current',
+            desc: 'Continuous Integration Build (latest in version control)',
+            path: 'http://build.fhir.org/ig/fhir/sushi-test',
+            status: 'ci-build',
+            current: true
+          },
+          {
+            version: '0.2.0',
+            fhirversion: '4.0.1',
+            date: '2019-09-01',
+            desc: 'Second STU ballot (Sep 2019 Ballot)',
+            path: 'http://hl7.org/fhir/sushi-test/stu2',
+            status: 'ballot',
+            sequence: 'STU 2'
+          },
+          {
+            version: '0.1.0',
+            fhirversion: '4.0.1',
+            date: '2019-05-01',
+            desc: 'Initial STU ballot (May 2019 Ballot)',
+            path: 'http://hl7.org/fhir/sushi-test/stu1',
+            status: 'ballot',
+            sequence: 'STU 1'
+          }
+        ]
+      });
+    });
+
+    it('should use the user-provided index.md if it exists', () => {
+      const indexPath = path.join(tempOut, 'input', 'pagecontent', 'index.md');
+      expect(fs.existsSync(indexPath)).toBeTruthy();
+      const content = fs.readFileSync(indexPath, 'utf8');
+      expect(content).toMatch('My special index page.');
+    });
+  });
+
+  describe('#invalid-data-ig', () => {
+    let pkg: Package;
+    let exporter: IGExporter;
+    let tempOut: string;
+
+    beforeAll(() => {
+      const fixtures = path.join(__dirname, 'fixtures', 'invalid-data-ig');
+      const config: Config = fs.readJSONSync(path.join(fixtures, 'package.json'));
+      pkg = new Package([], [], config);
+      exporter = new IGExporter(pkg, path.resolve(fixtures, 'ig-data'));
+      tempOut = temp.mkdirSync('sushi-test');
+      // No need to regenerate the IG on every test -- generate it once and inspect what you
+      // need to in the tests
+      exporter.export(tempOut);
+    });
+
+    afterAll(() => {
+      temp.cleanupSync();
+    });
+
+    it('should should log errors for invalid user-specified values in ig.ini', () => {
+      // Check for log messages indicating invalid input
+      expect(loggerSpy.getMessageAtIndex(-5)).toMatch(
+        /igi\.ini: sushi does not currently support overriding ig value\..*File: .*\/invalid-data-ig\/ig-data\/ig.ini/s
```

Comment:
This fails for me on Windows because it looks like you are checking for the "/" in the file path.

Reply:
Oops.